docs/release_process.rst

```diff
@@ -1,10 +1,9 @@
 Steps of the xaitk-saliency Release Process
 ===========================================
 Three types of releases are expected to occur:
-
-* patch
-* minor
-* major
+- major
+- minor
+- patch
```

Comment:
Is this supposed to render as an enumerated list? The different types of releases appear in-line in RTD.

Reply:
Oh darn, that was indeed supposed to be a bulleted list. Let me fix that.

Reply:
Done.